docs/source/guide/tutorials/base/deploy.rst

```diff
@@ -45,4 +55,125 @@ But it can be runned for any task in the same manner.
+    python -m pip install -r outputs/deploy/requirements.txt
+
+
+3. The following line will run the demo on your input source, 
+using the model in the ``model`` folder. You can pass as ``input`` a single image, a folder of images, a video file, or a web camera id.
+
+.. code-block::
+
+    (demo) ...$ python3 outputs/deploy/python/demo.py --input docs/utils/images/wgisd_dataset_sample.jpg \
+                                                      --models outputs/deploy/model
+
+You can press ``Q`` to stop inference during the demo running.
+
+The model inference on your custom image will look like this:
+
+.. image:: ../../../../utils/images/wgisd_pr_sample.jpg
+  :width: 600
+  :alt: this image shows the inference results on the WGISD dataset
+
+.. note::
+
+    If you provide a single image as input, the demo processes and renders it quickly, then exits. To continuously
+    visualize inference results on the screen, apply the ``loop`` option, which enforces processing a single image in a loop.
+    In this case, you can stop the demo by killing the process in the terminal (``Ctrl+C`` for Linux).
+
+To learn how to run the demo on Windows and MacOS, please refer to the ``outputs/deploy/python/README.md`` file in exportable code.
+
+4. To run a demo on a web camera, we need to know its ID. 
+We can check a list of camera devices by running this command line on Linux system:
+
+.. code-block::
+
+    sudo apt-get install v4l-utils
+    v4l2-ctl --list-devices
+
+The output will look like this:
+
+.. code-block::
+
+    Integrated Camera (usb-0000:00:1a.0-1.6):
+        /dev/video0
+
+After that, we can use this ``/dev/video0`` as a camera ID for ``--input``.
+
+5. We can also change ``config.json`` that specifies the confidence threshold and 
+color for each class visualization, but any changes should be made with caution. 
+
+For example, in our image of the winery we see, that a lot of objects weren't detected.
+The original confidence threshold was chosen based on the validation split results to maximize the final F1 metric, balancing precision and recall values. So, visual results can be not suitable enough for a user.
+To overcome this problem, we can decrease ``confidence_threshold`` in ``config.json file`` from **0.4** to **0.3**.
+
+.. code-block::
+
+    "model_parameters": {
+        "result_based_confidence_threshold": true,
+        "confidence_threshold": 0.3000000059604645,
+
+For visual purposes, we can also update the color of ``Chardonnay`` class from yellow to lilac to make it more distinguishable.
+
+.. code-block::
+
+    "all_labels": {
+        "0": {
+            "_id": "0",
+            "name": "Chardonnay",
+            "color": {
+                "red": 230,
+                 "green": 106,
+                 "blue": 226,
+
+The result will be the following:
+
+.. image:: ../../../../utils/images/wgisd_pr2_sample.jpg
+  :width: 600
+  :alt: this image shows the inference results on the WGISD dataset
+
+
```

Comment:
Please, add here something like: "Meanwhile this is example for object detection demo, other tasks can have their own tunable parameters that are also will be stored in ``config.json`` file"